collision.go

```diff
@@ -4,6 +4,7 @@ import (
 	"log"
 
 	"engo.io/ecs"
+	"github.com/luxengine/math"
```

Comment:
This error is related to the fact that `luxengine` uses fancy compiler-stuff for amd64 and i386 with the `Abs` method (the rest is fine). This is not related to the import path or function naming, but the implementation not being available for gopherjs.